rs/execution_environment/tests/storage_reservation.rs

```diff
@@ -650,6 +649,6 @@ fn reserved_cycles_memory_allocation_grow_to_full_capacity() {
 
     // The total amount of reserved cycles to claim the full subnet memory capacity
     // while reserving a lot of memory at once.
-    const EXPECTED_RESERVED_CYCLES: u128 = 41_449 * T;
+    const EXPECTED_RESERVED_CYCLES: u128 = 35_136 * T;
```

Comment:
The initial value of `pages` used to be `MAX_MEMORY_ALLOCATION` which is expressed in bytes and thus already much more than the maximum number of pages that can be successfully reserved. Now that the constant `MAX_MEMORY_ALLOCATION` no longer exists, let's use a simple large value which results in a different (coincidentally lower) number of reserved cycles. Still the number of reserved cycles here is much more than the number of reserved cycles in the test `reserved_cycles_stable_memory_grow_to_full_capacity` growing stable memory via small steps at a time.